.travis.yml

```diff
@@ -15,5 +15,6 @@ addons:
   apt:
     packages:
       - oracle-java8-installer
+      - oracle-java8-unlimited-jce-policy
```

Comment:
To support unlimited key size. Ref: https://github.com/joyent/java-manta/issues/240